### DISCUSSION TITLE
Looking for recommendations on setting-up multiple test environments.

### DISCUSSION THREAD

**Discussion by jramoyo-serenade** [swellstores/community#380] [Help] [answered]
We are looking to get recommendations from the community on the best way to setup multiple test environments.

Like most development teams, we have dedicated environments for Dev, Test, Staging and Production. 

Given that a Swell store only has 1 test environment available, how do most teams setup multiple non-production environments?

Also, what do they do when developing from their local machines?

**Comment by sags95** [comment, marked as answer]
@jramoyo-serenade Unfortunately plans only come with a test and live environment at the moment, but it will be possible to setup custom test environments in the future.
